modules/citeproc/citeproc.install

```diff
@@ -13,3 +13,29 @@ function citeproc_update_7100() {
     drupal_set_message(t('The "citeproc" module is now dependent on "libraries", and the "citeproc-php" library must be installed. See the README for more info.'), 'warning');
   }
 }
+
+/**
+ * Set citeproc version if you are updating.
+ * 
+ * That way the library used doesn't switch under your feet, when updating to 
+ * the latest version.
+ */
+function citeproc_update_7101() {
+  variable_set('citeproc_backend', 'citeproc_backend_v1');
+}
+
+/**
+ * Implements hook_install().
+ */
+function citeproc_install() {
+  module_load_include('inc', 'citeproc', 'includes/utilities');
+  $default = citeproc_v2_supported() ? CITEPROC_BACKEND_V2 : CITEPROC_BACKEND_V1;
```

Comment:
Not sure if the constants defined in .module will be present during bootstrap slimmed install hook. We have this issue in openseadragon. Errors only if installed for the first time. Can we live without constants? Also reviewing pulls from Minneapolis airport so i could be missing something! Resto looks good!!